llvm/lib/Target/AMDGPU/SIPreAllocateWWMRegs.cpp

```diff
@@ -70,6 +71,7 @@ class SIPreAllocateWWMRegsLegacy : public MachineFunctionPass {
     AU.addRequired<LiveIntervalsWrapperPass>();
     AU.addRequired<VirtRegMapWrapperLegacy>();
     AU.addRequired<LiveRegMatrixWrapperLegacy>();
+    AU.addRequired<MachineRegisterClassInfoWrapperPass>();
```

Comment:
Can you add a todo to update RCI with the additional reserved registers the pass sets